spec.html

```diff
@@ -28544,7 +28544,8 @@
             </tbody>
           </table>
         </figure>
-        <p>The production <emu-grammar>CharacterClassEscape :: `W`</emu-grammar> evaluates by returning the set of all characters not included in the set returned by <emu-grammar>CharacterClassEscape :: `w`</emu-grammar> .</p>
+        <p>and the Boolean *false*.</p>
```

Comment:
The markup around this area looks really weird (closing `</tbody>` without a matching start tag etc.) but this line, at least, is intentional. It’s the continuation of the sentence on line https://github.com/tc39/ecma262/pull/516/files/1944b451b767e30575150204d0d28a8ae270a3a5#diff-3540caefa502006d8a33cb1385720803L28315.